com.unity.render-pipelines.core/Runtime/Lighting/ProbeVolume/ProbeReferenceVolumeProfile.cs

```diff
@@ -11,29 +11,55 @@ namespace UnityEngine.Rendering
     /// </summary>
     public sealed class ProbeReferenceVolumeProfile : ScriptableObject
     {
-        /// <summary>
-        /// The default dimensions for APV's index data structure.
-        /// </summary>
-        public static Vector3Int s_DefaultIndexDimensions = new Vector3Int(1024, 64, 1024);
+        internal enum Version
+        {
+            Initial,
+        }
+
+        public enum CellSize
+        {
+            [InspectorName("1")]
+            CellSize1 = 1,
+            [InspectorName("9")]
+            CellSize9 = 9,
+            [InspectorName("27")]
+            CellSize27 = 27,
+            [InspectorName("81")]
+            CellSize81 = 81,
+            [InspectorName("243")]
+            CellSize243 = 243,
+        }
 
         /// <summary>
         /// The size of a Cell.
         /// </summary>
-        [Min(1)]
-        public int cellSize = 64;
+        public CellSize cellSizeInBricks = CellSize.CellSize81;
```

Comment:
I assume we didn't get any more solid info from artists about this right? We can go ahead with this, but given that I think is fairly sure that we'll change it, I'd add a comment so we don't rely on this being the representation for whatever reason.

Reply:
I am wondering already if we should not mention brick here at all (I'd like to keep that as hidden as possible from user), can we replace this with "cell size in probes" count for user facing stuff?

Reply:
Yeah, artists haven't decided yet, though I think they will decide to do something with a distance instead of this enum.
I'll add a comment, I can also change this to be a cell size in probes but that will be changed again when artists decide what they want

Reply:
Yeah let's sync on those after artists have their idea done, so I guess just a comment will do for now.